job-store-service/war/pom.xml

```diff
@@ -200,23 +222,23 @@
       <version>1.3-SNAPSHOT</version>
       <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>dk.dbc</groupId>
-      <artifactId>dataio-sink-types</artifactId>
-      <version>${project.version}</version>
-    </dependency>
+<!--    <dependency>-->
+<!--      <groupId>dk.dbc</groupId>-->
+<!--      <artifactId>dataio-sink-types</artifactId>-->
+<!--      <version>1.0-SNAPSHOT</version>-->
+<!--    </dependency>-->
```

Comment:
Ok? Hvad gør du i stedet?

Reply:
Tror at det var her AbstractMessageConsumerBean lå... og den har vi ikke brug for mere da vi ikke længere har consumere i bean form